### PR TITLE
feat(s3_checks): add several checks for s3 

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Context 

There were added 2 checks of S3 as well as the client.

### Description

Add checks extra718 and extra 763.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jfagoagas — APPROVED** [review]
Good job @sergargar !!! 🪣🪣🪣🪣

File: providers/aws/services/s3/s3_service.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            threads.append(threading.Thread(target=call, args=(regional_client,)))
+        for t in threads:
+            t.start()
+        for t in threads:
+            t.join()
+
+    def __list_buckets__(self, regional_client):
+        logger.info("S3 - Listing buckets...")
+        try:
+            list_buckets = regional_client.list_buckets()
```
Comment (jfagoagas) [inline providers/aws/services/s3/s3_service.py:34]: Why if you query AWS only once to retrieve the bucket list?

**Review by jfagoagas — CHANGES_REQUESTED** [review]
Check my comment please

File: providers/aws/services/s3/s3_service.py (hunk 1 above)
Comment (MrCloudSec) [reply providers/aws/services/s3/s3_service.py:34]: At the moment, the only solution to speed up this algorithm is to have regional resources, which is going to be done in the future.

**Review by jfagoagas — APPROVED** [review]
🚀  awesome!

**Pull request merged**